CentralComputing/Pod.cpp

```diff
@@ -5,26 +5,32 @@
 using namespace std;
 
 shared_ptr<Pod_State> state_machine;
-SafeQueue<shared_ptr<NetworkManager::Network_Command>> command_queue;
 bool running = true;
```

Comment:
This `running` overshadows the atomic<bool> that the network manager defines, right?

Reply:
Kind of? I'll rework it a little better, I need to add better closing/ctrl-c behavior